src/sonic-yang-models/tests/yang_model_tests/tests/restapi.json

```diff
@@ -0,0 +1,9 @@
+{
+    "RESTAPI_TABLE_WITH_INCORRECT_CERT": {
+        "desc": "TABLE_WITH_INCORRECT_CERT failure.",
+        "eStrKey": "Pattern"
```

Comment:
"eStr": ["server_crt"]

This way, Libyang error will be matched against only server_crt field.